src/agent/src/rpc.rs

```diff
@@ -1291,7 +1328,10 @@ fn get_memory_info(block_size: bool, hotplug: bool) -> Result<(u64, bool)> {
                 info!(
                     sl!(),
                     "hotplug memory error: {}",
-                    e.as_errno().unwrap().desc()
+                    e.as_errno()
+                        .ok_or("Unable to extract errno")
+                        .map_err(|e| anyhow!(e))?
+                        .desc()
```

Comment:
Hmm... using `as_errno()` seems bogus in the first place.  We're just trying to display the error in the log message, so why not just... display the error in the log message rather than (fallibly) converting to an errno and back again.